installer/main.sh

```diff
@@ -138,12 +138,16 @@ done
 shift "$((OPTIND-1))"
 
 # Check against the minimum version of Chromium OS
-if ! awk -F= '/_RELEASE_BUILD_NUMBER=/ { exit int($2) < '"${CROS_MIN_VERS:-0}"' }' \
+if [ -n "$DOWNLOADONLY" ]; then
+    :
+elif ! grep -q '_RELEASE_BUILD_NUMBER=' /etc/lsb-release 2>/dev/null; then
```

Comment:
Just curious, is this enough or should it be even more qualified with 'CHROMEOS_RELEASE_BUILD_NUMBER' here and on line 145?

Reply:
I think that would mess up Chromium OS

Reply:
Well, I thought so too at first but after checking it seems all the Chromium OS derivatives use the 'CHROMEOS_' prefix. Here's the `lsb-release` files for CrOS, Chromium OS, Cloudready, & Flint OS (yes, I have all of them)

- [lsb-release.all.txt](https://transfer.sh/vwTCd/lsb-release.all.txt)

But I guess things can change so whatever you decide is fine with me.